packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/FieldGuideContainer.js

```diff
@@ -45,8 +42,6 @@ function FieldGuideContainer ({
   const modalPropsToUse = (size === 'small') ? modalProps : Object.assign({}, modalProps, { rndProps })
   return (
     <FieldGuide
-      boxHeight={boxHeight}
-      boxWidth={boxWidth}
```

Comment:
How is the size limited now, so that the field guide doesn't get in the way while you classify?

Reply:
Was that something you experienced while running this branch locally?

The FieldGuideContainer sets minHeight and minWidth, which are passed to the Modal component.
https://github.com/zooniverse/front-end-monorepo/blob/a6f560a1e42ceea24bac0192f5365eb39d2851c7/packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/FieldGuideContainer.js#L15-L16

Those values remain in this PR and set the modal dimensions for desktop viewports.

Reply:
👍🏻 cool! I'm just trying to understand the code changes here.

On desktop, what I'm seeing is that the width of the field guide initially overflows the edge of the viewport, cutting off half the text.

<img width="586" alt="The PH-TESS field guide open but with half the text cut off by the righthand side of the viewport." src="https://user-images.githubusercontent.com/59547/214816577-c6812806-7385-45fb-97a0-48a4ceee941f.png">

Reply:
That bug pushes the Close button out of the viewport too. I think the content width isn't set properly until after all the images have loaded, so this could be a Markdown bug rather than a Field Guide bug.

Reply:
The question about the guide being open, but not in the way, while you classify, is from this discussion: https://github.com/zooniverse/front-end-monorepo/discussions/2424

The changes here look fine to me. It would be good to get that width bug fixed when the content first loads.

Reply:
Agreed - it's definitely bad UX that the close button is pushed out of the viewport. You're right it's a result of the way we handled Media in Markdownz. For instance, because PH-TESS's markdown image tags do not include any custom dismensions, yet the FieldGuideContainer is positioned absolutely with a fixed width.